spec/support/controller_macros.rb

```diff
@@ -0,0 +1,17 @@
+module ControllerMacros
+  def login_user
```

Comment:
Nie prościej zwinąć to do jednej funkcji, usera tworzyć poprzez `let(:user)` dla każdego testu z osobna i w beforze przed testami tego usera logować?